internal/agent/device/spec/manager.go

```diff
@@ -103,10 +109,21 @@ func (s *manager) Initialize(ctx context.Context) error {
 	}
 	// reconcile the initial spec even though its empty
 	s.queue.Add(ctx, newVersionedDevice("0"))
+
+	// Audit the bootstrap event
+	s.logAudit(ctx, nil, "", "0", audit.AuditResultSuccess, audit.AuditReasonBootstrap, audit.AuditTypeCurrent)
```

Comment:
the audit log is enabling auditing of the state transitions of the on disk representations of device spec. what state change takes place here? if none should it be a log?